### ISSUE TITLE
Make product page breadcrumbs closer to Collection titles

### ISSUE THREAD

**Issue by erinemay** [iFixit/react-commerce#1104]
I know there was some intentional work done on the product page breadcrumbs resulting in an imperfect match between collection and product breadcrumbs. But these are a little weird. It looks like they're matching the urls which aren't always the best text representation of the page. 

Breadcrumbs on the product page: 
![image](https://user-images.githubusercontent.com/72166715/205775340-bcfd9117-025f-4c08-83ec-6200d4703c10.png)

These two sound like bad translations:
- "Samsung Galaxy Phone Note" should be [Samsung Galaxy Note](https://www.ifixit.com/Parts/Samsung_Galaxy_Phone_Note)
- "Samsung Android Phone" should be [Samsung Phone](https://react-commerce.vercel.app/Parts/Samsung_Android_Phone)

The "Plus" is represented by a "+" everywhere else on the site except for URLs
- "Galaxy Note10 Plus" should be [Samsung Galaxy Note10+](https://react-commerce.vercel.app/Parts/Galaxy_Note10_Plus)

**Comment by dhmacs** [comment]
Is this only reporting data issues, or is any code change required?

**Comment by erinemay** [comment]
I think it's a code change. The equivalent collection page breadcrumbs are just fine. Can we just match them without "Parts"?
https://www.ifixit.com/Parts/Galaxy_Note10_Plus
![image](https://user-images.githubusercontent.com/72166715/205982515-a6550833-cce4-454c-b544-46100353e53c.png)

**Comment by dhmacs** [comment]
After reviewing the code, it appears that the issue is likely related to the data. We're not doing anything fancy, we're just taking breadcrumbs from the metafield and displaying them here. So perhaps the code that syncs the metafield is not using the right attribute?

**Comment by sterlinghirsh** [comment]
Yeah the page was created with the wrong title. The display title makes it look better, but we're inconsistent about using the display title. I'm calling this a data issue.

**Comment by sterlinghirsh** [comment]
But I think there is something odd to the idea that the breadcrumbs of the product page are device titles instead of product lists. Gonna think on this one.
